plugin.video.vrt.nu/resources/lib/helperobjects/helperobjects.py

```diff
@@ -1,10 +1,7 @@
-# -*- coding: UTF-8 -*-
+# -*- coding: utf-8 -*-
 
 # GNU General Public License v2.0 (see COPYING or https://www.gnu.org/licenses/gpl-2.0.txt)
 
-from __future__ import absolute_import, division, print_function, unicode_literals
```

Comment:
We definitely don't need `print_function`. I'm not against your point of view. But I rather like a faster addon when possible.

Reply:
Indeed, we never need print_function. This was originally added since Tox hints for it.
But we disabled those tests, as it warns even when imported :-)

Reply:
I doubt this will make a big difference in performance, but removed them anyway :-)